edx_django_utils/management/docs/decisions/0001-user-and-group-management-commands.rst

```diff
@@ -0,0 +1,20 @@
+User and Group Management Commands
+===============================
+
+Context
+-------
+
+Open edX administrators ocassionally need to grant permissions to users for certain django services (e.g. ecommerce, registrar). Right now, this is done manually via django admin of the django app. This method of granting permissions is not endorsed.
+
+The user and group management commands in edx-platform, are supposedly generic enough to be used via a common library/app so that users and groups can be managed without much (repeated) modificaiton to IDAs.
+
+Decision
+--------
+
+User and group management commands should be moved to edx-django-utils (this repository) from edx-platform and edx_django_utils should be added to INSTALLED_APPS section of all django apps so that the user and group management commands are available to the apps.
+
+Kyle's notes can be found `here https://github.com/edx/app-permissions/pull/1454/files#diff-ab850bbacf01a93fe03d0d87d0ed5d2db2f1e1d1d27c7057eb556e63b084c4e7R7-R48`.
+
+Consequences
+------------
+
```

Comment:
Maybe add a note about the plans for adding this to others services (IDAs)? Will that be left to the owning teams? Do you plan on updating one example service?

Reply:
Hey Rob, that's a good point about adding that context.  We currently have a ticket to investigate which services would benefit from using app-permissions and have not determined if SRE will have capacity to support that work, however I agree that a non-edxapp example or a guide would be useful and know that ecommerce and registrar are both services that we would like to see using app-permissions.